portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,23 +28,30 @@
 /** Servlet that returns some example content. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  private final List<String> myComments = new List<>();
```

Comment:
I don't think this will build since `List` is an interface and can't be instantiated. You could switch to `new ArrayList<>()` to solve this problem.

Reply:
thanks! updated!